pygsti/algorithms/germselection.py

```diff
@@ -147,6 +145,42 @@ def find_germs(target_model, randomize=True, randomization_strength=1e-2,
     toss_random_frac : float, optional
         If specified this is a number between 0 and 1 that indicates the random fraction of candidate
         germs to drop randomly following the deduping procedure.
+        
+    mode : str, optional (default 'allJac')
```

Comment:
Should change this doc line to:
`mode : {'allJac', 'singleJac', 'compactEVD'}, optional`
This shows the options clearly while also communicating that it must be a string.  No need to put default on this line, as it's in the function definition and it adds one more place to get things out of sync.

Reply:
Done